RecoTracker/MkFitCore/src/PropagationMPlex.icc

```diff
@@ -235,48 +259,66 @@ inline void parsAndErrPropFromPathL_impl(const Tf& __restrict__ inPar,
 
   //now we need jacobians to convert to/from curvilinear and CCS
   // code from TrackState::jacobianCCSToCurvilinear
-  MPlex56 jacCCS2Curv;
-  for (int n = nmin; n < nmax; ++n) {
-    for (int ii = 0; ii < 5; ii++) {
-      for (int jj = 0; jj < 6; jj++) {
-        jacCCS2Curv(n, ii, jj) = 0.f;
-      }
-    }
-    jacCCS2Curv(n, 0, 3) = inChg(n, 0, 0) * sinT[n - nmin];
-    jacCCS2Curv(n, 0, 5) = inChg(n, 0, 0) * cosT[n - nmin] * inPar(n, 3, 0);
-    jacCCS2Curv(n, 1, 5) = -1.f;
-    jacCCS2Curv(n, 2, 4) = 1.f;
-    jacCCS2Curv(n, 3, 0) = -sinPin[n - nmin];
-    jacCCS2Curv(n, 3, 1) = cosPin[n - nmin];
-    jacCCS2Curv(n, 4, 0) = -cosPin[n - nmin] * cosT[n - nmin];
-    jacCCS2Curv(n, 4, 1) = -sinPin[n - nmin] * cosT[n - nmin];
-    jacCCS2Curv(n, 4, 2) = sinT[n - nmin];
-  }
+  MPlex56 jacCCS2Curv(0.0f);
+  jacCCS2Curv.aij(0, 3) = mpt::negate_if_ltz(sinT, inChg);
+  jacCCS2Curv.aij(0, 5) = mpt::negate_if_ltz(cosT, inChg);
```

Comment:
source in `TrackState::jacobianCCSToCurvilinear` has `    jac(0, 5) = charge * cosT * invpt;`

Reply:
> source in `TrackState::jacobianCCSToCurvilinear` has ` jac(0, 5) = charge * cosT * invpt;`

@cerati which one do you think is a bug?

Reply:
I am not familiar with `mpt::negate_if_ltz`... maybe @osschar ?

Reply:
I think my version before this change was the same as `TrackState::jacobianCCSToCurvilinear`

Reply:
my point is that the `1/pT` is missing

Reply:
it was not missing in my version (you can see it if you look at the code that was removed)

Reply:
oh rats, it's my fault again :)
or maybe it's better because of it ;)